frontend/src/pages/Login/Login.tsx

```diff
@@ -49,13 +48,6 @@ export const Login: FC<Props> = () => {
 
       <RegisterText> ────── OR ──────</RegisterText>
 
-      {/* <SocialButton
```

Comment:
We shouldn't have commented out code - that's not actually what comments are for 😉 
We use git so that we don't have to do this - if we want to find this bit of code again, all we need to do is navigate back to the commit/ branch it was on